variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "memory_size" {
+  default     = 512
```

Comment:
lotta memory for a little lambda, no?

Reply:
I just grabbed the setting from upstream

Reply:
memory set vs memory used:
`512 / 67`

I can definitely reduce